examples/socket-mode/app.js

```diff
@@ -184,3 +184,105 @@ app.command('/socketslash', async ({ command, ack, say }) => {
 
   await say(`${command.text}`);
 });
+
```

Comment:
I don't think this is the best place for this 😅 

This example showcases the most used features of slack, for example it doesn't include custom steps or link unfurls so I don't think we should put work objects here 

A dedicated sample in https://github.com/slack-samples may be a better place for this 🤔 
I also heard that @zimeg may be working on a repository in https://github.com/slack-samples where we can showcase all of slacks features, but I'm not sure if there is a timeline for this

Reply:
Fair! I'll remove it from this PR. Maybe we can just show sample code snippets in the docs for now